apis/swagger.yml

```diff
@@ -351,6 +351,28 @@ paths:
           schema:
            $ref: "#/definitions/Error"
       tags: ["Container"]
+  /containers/{id}/pause:
+    post:
+      summary: "Pause a container"
+      operationId: "ContainerPause"
+      parameters:
+        - name : "id"
+          in : "path"
+          required: true
+          description: "ID or name of the container"
+          type: "string"
+      responses:
+        200:
```

Comment:
Still here, I think we should keep 204 rather than 200. Since we need to be compatible with Moby's API. See: https://github.com/moby/moby/blob/master/api/swagger.yaml#L5454